python/pyarrow/tests/test_pandas.py

```diff
@@ -2040,6 +2040,18 @@ def test_strided_objects(self, tmpdir):
         # This yields strided objects
         df = pd.DataFrame.from_dict(data)
         _check_pandas_roundtrip(df)
+    
```

Comment:
```suggestion

```

Reply:
Sorry, tried pushing with the GitHub UI suggested change but it removed the line. I've pushed a second commit manually with the correct fix.